demo/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+    entry: './src/libs.js',
+    output: {
+        filename: 'index.js',
+        path: path.resolve(__dirname, 'libs/')
+    },
+    performance: {
+        hints: 'warning'
+    },
+    optimization: {
+        minimize: false
+    },
+    plugins: [
+        // clean out demo/libs/ before building
+        new CleanWebpackPlugin()
```

Comment:
**Question**: When I was setting these libraries up with Webpack I noticed that there is a property you can pass to output called `clean` but I haven't looked up the pros/cons of each. Might be worth looking into if we could replace with the property to remove that as a dependency for these packages (and bedrock too)

Reply:
Oh nice, let’s look into this and see if we can simplify a few projects using it 🙂